log.txt

```diff
@@ -0,0 +1,64 @@
+Read from XML. Action successful.

```

Comment:
Nie potrzebujemy tego pliku, utworz plik .gitignore do ktorego dopiszesz ten plik, pozniej go usun - to spowoduje ze nie bedzie go w Pull Requestcie.